src/components/UI/Card/Card.tsx

```diff
@@ -79,44 +78,24 @@ export const Card: FC<CardProps> = ({
           />
         </div>
       )}
-      {priceType && (
+      {cardType === "priceType" && (
         <div className={styles["card__price-box"]}>
           <div className={styles["card__price"]}>
             {currency} {price}
           </div>
-          {priceChange > 0 && (
-            <div
-              className={`${styles["card__price-change"]} ${styles["success"]}`}
-            >
-              +{priceChange}%
-            </div>
-          )}
-          {priceChange < 0 && (
-            <div
-              className={`${styles["card__price-change"]} ${styles["danger"]}`}
-            >
-              {priceChange}%
-            </div>
-          )}
+          <IncreaseOrDecrease
+            procent={priceChange}
+            className={styles["card__price-change"]}
+          />
         </div>
       )}
-      {userType && (
+      {cardType === "userType" && (
         <div className={styles["card__price-box"]}>
           <div className={styles["card__price"]}>${`00.00`}</div>
-          {priceChange > 0 && (
-            <div
-              className={`${styles["card__price-change"]} ${styles["success"]}`}
-            >
-              +{`00.00`}%
-            </div>
-          )}
-          {priceChange < 0 && (
-            <div
-              className={`${styles["card__price-change"]} ${styles["danger"]}`}
-            >
-              {`00.00`}%
-            </div>
-          )}
+          <IncreaseOrDecrease
+            number={"00.00"}
```

Comment:
```suggestion
            number="00.00"
```
но опять же (про что выше писал) не совсем понимаю, что это за нули